analytics/spec/events/video_played.yaml

```diff
@@ -12,7 +12,7 @@ rules:
         video_url:
           description: A URL identifier for the video
           type: string
-        progress:
+        video_progress:
```

Comment:
The spec defines properties `{ video_url, progress }`, but also notes the required properties as `{ video_url, video_progress }`. I think `progress` is a minor slip and should be `video_progress` for consistency, so have updated accordingly. cc @BrandonRomano